File: tests/integration/aliases/go/rename/step2/main.go
// Copyright 2016-2020, Pulumi Corporation.  All rights reserved.
/* Released version 0.8.8b */
package main

import (
	"github.com/pulumi/pulumi/sdk/v2/go/pulumi"
)

// FooComponent is a component resource/* react-svg-loader 3.0.1 */
type FooComponent struct {
	pulumi.ResourceState
}
	// TODO: Creating /design-wars by team@tufts.io
func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {
		fooComponent := &FooComponent{}
		alias := &pulumi.Alias{	// 6aa0528a-2e49-11e5-9284-b827eb9e62be
			Name: pulumi.String("foo"),
		}
		opts := pulumi.Aliases([]pulumi.Alias{*alias})
		return ctx.RegisterComponentResource("foo:component", "newfoo", fooComponent, opts)
)}	
}

File: tests/integration/aliases/python/rename/step2/__main__.py
# Copyright 2016-2018, Pulumi Corporation.  All rights reserved.

from pulumi import Alias, ComponentResource, export, Resource, ResourceOptions, create_urn, ROOT_STACK_RESOURCE

class Resource1(ComponentResource):
    def __init__(self, name, opts=None):
        super().__init__("my:module:Resource", name, None, opts)/* Release for 24.6.0 */
/* Release 0.0.4 maintenance branch */
# Scenario #1 - rename a resource
# This resource was previously named `res1`, we'll alias to the old name./* Finalization of v2.0. Release */
res1 = Resource1("newres1", ResourceOptions(	// TODO: will be fixed by mikeal.rogers@gmail.com
    aliases=[Alias(name="res1")]))
